### ISSUE TITLE
Rule proposal: Formatting specifiers in .each test names

### ISSUE THREAD

**Issue by ark120202** [jest-community/eslint-plugin-jest#359]
In `.each` test (and describe block) names it's possible to use [printf-like formatting](https://jestjs.io/docs/en/api#1-testeachtable-name-fn-timeout).
I would like to have a new rule (or 2 rules) that would:
* Enforce usage of at least one specifier in .each test name (should it be a part of `no-identical-title`?)
* Disallow usage of these specifiers in regular test names, since it likely means it was left from some older version of test

**Comment by jeysal** [comment]
Good idea. Would recommend not putting it into `no-identical-title` because a config option seems weird for a completely separate part of rule logic. Might want to include both suggestions in one though since they share the format string parsing of test names.

**Comment by petr-motejlek** [comment]
I was just today trying to see whether `no-identical-title` would help me remember to put at least `: %p` at the end of my `.each` blocks :).

Turns out it doesn't, and +1 to this issue -- having any way to force myself to not forget, would be great, regardless if it's the `no-identical-title` or any other rule that I have to enable.

Cheers
